packages/router-lite/src/route-context.ts

```diff
@@ -186,6 +190,7 @@ export class RouteContext {
     this._moduleLoader = parentContainer.get(IModuleLoader);
 
     const container = this.container = parentContainer.createChild();
+    this._platform = container.get(IPlatform);
```

Comment:
```suggestion
    const platform = this._platform = container.get(IPlatform);
```
though it seems we don't use `_platform`?

Reply:
We have not used the platform in the ctor but in `_createComponentAgent`.
![image](https://github.com/user-attachments/assets/d7ef6b89-7554-490a-afe1-ebdd6fec9831)